data/json/items/armor/ammo_pouch.json

```
@@ -507,5 +507,47 @@
     ],
     "use_action": { "type": "holster", "holster_prompt": "Stash ammo", "holster_msg": "You stash your %s." },
     "flags": [ "OVERSIZE", "STURDY", "OUTER" ]
+  },
+  {
+    "id": "longboot_quiver",
+    "type": "ARMOR",
+    "name": { "str": "thigh-high boot quiver" },
+    "description": "A rather comical 'quiver' worn at the waist that can hold 40 arrows or bolts.  Use insert to store arrows or bolts.",
+    "weight": "1540 g",
+    "volume": "1000 ml",
+    "price": 5000,
+    "price_postapoc": 1000,
+    "bashing": 2,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "looks_like": "boots_larmor",
+    "color": "black",
+    "covers": [ "LEG_EITHER" ],
+    "coverage": 10,
+    "encumbrance": 16,
+    "material_thickness": 1,
+    "pocket_data": [ { "ammo_restriction": { "arrow": 40, "bolt": 40 }, "moves": 25 } ],
```

```suggestion
    "pocket_data": [ { "ammo_restriction": { "arrow": 40, "bolt": 40 }, "moves": 25 } ],
```
Sources on historic quivers suggest between 20 and 30 arrows in a made for arrows quiver.  I think we should assume significantly less for a makeshift quiver.